release/xgboost_tests/oss_cluster_cpu_small.yaml

```diff
@@ -0,0 +1,44 @@
+cluster_name: ray-xgboost-release-cpu-small
+
+max_workers: 4
+
+upscaling_speed: 32
+
+idle_timeout_minutes: 15
+
+docker:
+    image: rayproject/ray-ml:1.2.0
+    container_name: ray_container
+    pull_before_run: true
+
+provider:
+    type: aws
+    region: us-west-2
+    availability_zone: us-west-2a
+    cache_stopped_nodes: false
+
+available_node_types:
+    cpu_4_ondemand:
+        node_config:
+            InstanceType: m5.xlarge
+        resources: {"CPU": 4}
+        min_workers: 3
+        max_workers: 3
+
+auth:
+    ssh_user: ubuntu
+
+head_node_type: cpu_4_ondemand
+worker_default_node_type: cpu_4_ondemand
+
+file_mounts: {
+    "~/release-automation-xgboost_tests": "."
+}
+
+setup_commands:
+    - pip install pytest xgboost_ray
+    - sudo mkdir -p /data || true
+    - sudo chown ray:1000 /data || true
```

Comment:
Needs to `sudo chown ray / `because the fault tolerance tests try to write to root.